packages/dds/merge-tree/src/client.ts

```diff
@@ -996,18 +997,46 @@ export class Client {
         }
     }
 
-    getContainingSegment<T extends ISegment>(pos: number, op?: ISequencedDocumentMessage) {
-        let seq: number;
-        let clientId: number;
+    private getRefSeqAndClientId(op?: ISequencedDocumentMessage) {
```

Comment:
this is an similar existing method: getClientSequenceArgs. should be resuable here